CMakeLists.txt

```diff
@@ -90,8 +85,10 @@ set(MCNP2CAD_PUB_HEADERS dataref.hpp
                          mcnp2cad.hpp
                          MCNPInput.hpp
                          options.hpp
-                         version.hpp
```

Comment:
don't we still want this to be installed even though it is autogenerated?

Reply:
we do at line 91

Reply:
I think ?!

Reply:
I think line 91 makes it available for the build, but doesn't install it for use by downstream applications, although I'm not 100% sure.

Reply:
yes you are probably right...